D: 5.4
xOffset: 0.6
yOffset: 0.4
zOffset: 0
var: true
var_gain: 100
sim: true